rust/connlib/snownet/src/node.rs

```diff
@@ -276,6 +276,15 @@ where
                 &mut self.pending_events,
             );
 
+            // Initiate a new WG session.
+            //
+            // We can have up to 8 concurrent WireGuard sessions in boringtun before the oldest one get overwritten.
```

Comment:
Does this mean a client can only connect to 8 gateways? Or does mean up to 8 sessions _per endpoint_? Assuming it's the latter but just double-checking.

Reply:
This is all per-connection.